scripts/bfasst/compare_waveforms/File_Generation/waveform_generator.py

```diff
@@ -24,14 +21,12 @@ def generate_VCD(paths, i):
     gtkwave(paths, i)
 
 
-"""Creates a waveform that only takes into account the inputs/outputs (totally ignores all internal components) and then
-takes the input/output graph and turns it into a VCD file to be compared against."""
-
-
 def gtkwave(paths, i):
```

Comment:
Given that you switched the depth of the vcd generation from all signals to only the top level,  does gtkwave still help?